Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM python:3.10-alpine3.16
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PYTHONUNBUFFERED=1
+# RUN apt-get update -y && apt-get upgrade -y
+WORKDIR /app
+COPY requirements.txt requirements.txt
+RUN pip3 install --upgrade pip && pip3 install -r requirements.txt
+COPY . .
+EXPOSE 5000
+CMD ["python3", "app.py"]
```

Comment:
optional: would be nice if you switch user after installing all deps

Reply:
Resolved with commit e01653cd51757d452ac3c6c6815815c4e8717415